mailpoet/assets/css/src/components-public/captcha.scss

```diff
@@ -10,3 +10,10 @@
     }
   }
 }
+
+// WP registration form
+form#registerform .g-recaptcha:not([data-size='invisible']) {
```

Comment:
Good point. I think so.

There’s a chance of conflict if both our reCAPTCHA and another solution are active at the same time. However, if the MP solution is turned off, it won’t load any CSS or JS on the WP registration form, so there shouldn’t be any issues for other plugins acting on the same form.

When both our solution and another one are active at the same time, that would likely break both rendering and validation. It's not normal to have two reCAPTCHA solution on the same form. I didn’t address the conflict issue intentionally because there are so many options out there. I’d consider it a user mistake—it’s easy to fix by disabling one of the solutions.